packages/cadl-ranch-specs/http/resiliency/package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
shouldn't need that file, the one at the root of `cadl-ranch-specs` should cover all the scenarios